src/store/actions/index.js

```diff
@@ -1,8 +1,14 @@
-import { ADD_TASK } from "store/constants";
+import { ADD_TASK, DELETE_TASK } from "store/constants";
 
 export const addTask = (task) => {
   return {
     type: ADD_TASK,
     payload: task,
   };
 };
+export const deleteTask = (index) => {
```

Comment:
separate export statements by line breaks

```suggestion
};

export const deleteTask = (index) => {
```